shell_scripts/pretty-macro.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
```

Comment:
Could you put this in top level scripts/ directory

Reply:
Done.